src/freenet/node/useralerts/N2NTMUserAlert.java

```diff
@@ -106,21 +116,55 @@ private String l10n(String key, String pattern, String value) {
 	@Override
 	public void onDismiss() {
 		DarknetPeerNode pn = (DarknetPeerNode) peerRef.get();
-		if(pn != null)
+		if (pn != null)
 			pn.deleteExtraPeerDataFile(fileNumber);
 	}
 
 	@Override
 	public FCPMessage getFCPMessage() {
-		return new TextFeedMessage(getTitle(),
-				getShortText(), getText(), getPriorityClass(), getUpdatedTime(), sourceNodeName,
-				composedTime, sentTime, receivedTime, messageText);
+		return new TextFeedMessage(getTitle(), getShortText(), getText(), getPriorityClass(), getUpdatedTime(),
+				sourceNodeName, composedTime, sentTime, receivedTime, messageText);
 	}
 
-    @Override
-    public long getUpdatedTime() {
-        return receivedTime;
-    }
+	@Override
+	public long getUpdatedTime() {
+		return receivedTime;
+	}
+
+	/**
+	 * @return the messageText
+	 */
+	public String getMessageText() {
+		return messageText;
+	}
+
+	/**
+	 * @return the fileNumber
+	 */
+	public int getFileNumber() {
+		return fileNumber;
+	}
+
+	/**
+	 * @return the composedTime
+	 */
+	public long getComposedTime() {
+		return composedTime;
+	}
+
+	/**
+	 * @param composedTime the composedTime to set
```

Comment:
This seems to be in the wrong place. :)

Reply:
- The entropy of all the JavaDocs in this areas is very low anyway, they only repeat what's in the function names, we could save the space for something which really needs documentation.

- For docs which fit into one line I'd format them like this to require less scrolling:
```java
/** Some documentation */
public void function() {
```

Reply:
@Bombe argl, leftovers from removing the setters … thanks :)

@xor-freenet yes, they can all go.